sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -409,7 +422,7 @@ object QueryTest {
   }
 
   def checkAnswer(df: DataFrame, expectedAnswer: java.util.List[Row]): String = {
```

Comment:
If we would like to differentiate the method name - `checkAnswer` for calling `fail` and `getErrorMessageInCheckAnswer` for getting message, this method should be changed as well.

Reply:
Make sense